homeassistant/components/switchbot/__init__.py

```diff
@@ -110,7 +110,9 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     )
     entry.async_on_unload(coordinator.async_start())
     if not await coordinator.async_wait_ready():
-        raise ConfigEntryNotReady(f"Switchbot {sensor_type} with {address} not ready")
+        raise ConfigEntryNotReady(
+            f"Switchbot {sensor_type} with {address} is not advertising state"
```

Comment:
How will the message read? Sounds like some words are missing.

Reply:
I think it can be simplified.  We already know it is a switchbot and the name of the device.

The only thing useful for troubleshooting here is the address.

I'll reduce it down.

Reply:
<img width="262" alt="Screen Shot 2022-09-17 at 3 55 24 AM" src="https://user-images.githubusercontent.com/663432/190848928-270645ca-47ee-4848-9936-a0d94524f5eb.png">

Reply:
Adjusted in 77045eb995ebef582138acda8df4b6721ca6d82f